libs/ledgerjs/packages/types-live/src/transaction.ts

```diff
@@ -13,9 +13,9 @@ export type SignedOperation = {
   // usually the device signature hex OR anything that is needed to broadcast (can be an inline JSON)
   signature: string;
   // sometimes a coin needs the raw object (it must be serializable)
-  signatureRaw?: Record<string, any>;
+  signatureRaw?: Record<string, unknown>;
```

Comment:
We can't have a more precise type for signatureRaw ? The value is different everytime ?

Reply:
Yes correct, it's basically a free field to provide any additional data, which is used differently by different families.

I've renamed it to a more generic `rawData` field.